spec/player_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe Player do
+  describe '#initialize' do
+    it 'exists' do
+      player = Player.new
+
+      expect(player).to be_a(Player)
+      expect(player.symbol).to be_a(String)
```

Comment:
What do you think about expecting `player.symbol` to eq `'X'`? Would we need to specify if the symbol is a string of the letter/character we want? Curious about your thoughts!

Reply:
Noted! I think checking for "X" is a better test.

Reply:
Sounds good! Will resolve!